pkg/kv/kvserver/replica_consistency_test.go

```diff
@@ -118,3 +124,100 @@ func TestGetChecksumNotSuccessfulExitConditions(t *testing.T) {
 	}
 	require.Nil(t, rc.Checksum)
 }
+
+// TestReplicaChecksumSHA512 checks that a given dataset produces the expected
+// checksum.
+func TestReplicaChecksumSHA512(t *testing.T) {
```

Comment:
Did you figure it out? What was it?

Reply:
Not really. I gave up trying to use a running server for this, and just call directly into `sha512()` giving it a fixed Pebble database. That's really all I'm interested in testing here anyway, so that's fine.